sa.kubeconfig

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
I think we should be adding this file to gitignore as it was auto generated.

Reply:
yea agreed. We can remove it later.